### PR TITLE
MyJansson small refactoring.

### PR DESCRIPTION
**Pull request by mgsergio**
@ygorshenin

### CODE REVIEW

File: 3party/jansson/myjansson.hpp [hunk 1]
```diff
@@ -24,6 +24,10 @@ using JSONPtr = std::unique_ptr<json_t, JSONDecRef>;
 inline JSONPtr NewJSONObject() { return JSONPtr(json_object()); }
 inline JSONPtr NewJSONArray() { return JSONPtr(json_array()); }
 inline JSONPtr NewJSONString(std::string const & s) { return JSONPtr(json_string(s.c_str())); }
+inline JSONPtr NewJSONInt(json_int_t value) { return JSONPtr(json_integer(value)); }
+inline JSONPtr NewJSONReal(double value) { return JSONPtr(json_real(value)); }
+inline JSONPtr NewJSONBool(bool value) { return JSONPtr(value ? json_true() : json_false()); }
+inline JSONPtr NewJSONull() { return JSONPtr(json_null()); }
```
Comment (ygorshenin) [inline 3party/jansson/myjansson.hpp]: NewJSONNull

File: 3party/jansson/myjansson.hpp [hunk 2]
```diff
@@ -24,6 +24,10 @@ using JSONPtr = std::unique_ptr<json_t, JSONDecRef>;
 inline JSONPtr NewJSONObject() { return JSONPtr(json_object()); }
 inline JSONPtr NewJSONArray() { return JSONPtr(json_array()); }
 inline JSONPtr NewJSONString(std::string const & s) { return JSONPtr(json_string(s.c_str())); }
+inline JSONPtr NewJSONInt(json_int_t value) { return JSONPtr(json_integer(value)); }
+inline JSONPtr NewJSONReal(double value) { return JSONPtr(json_real(value)); }
```
Comment (ygorshenin) [inline 3party/jansson/myjansson.hpp]: NewJSONDouble

Comment (ygorshenin) [inline 3party/jansson/myjansson.hpp]: Ah, OK.

File: 3party/jansson/myjansson.hpp (hunk 1 above)
Comment (mgsergio) [inline 3party/jansson/myjansson.hpp]: Done.

**Comment by mgsergio** [comment]
PTAL, DNM

File: 3party/jansson/myjansson.cpp [hunk 3]
```diff
@@ -54,120 +24,34 @@ json_t * GetJSONOptionalField(json_t * root, std::string const & field)
...
-  {
-    result = def;
-    return;
-  }
-  if (!json_is_boolean(val))
-    MYTHROW(my::Json::Exception, ("The field", field, "must contain a boolean value."));
-  result = json_is_true(val);
+  if (!json_is_true(root) && !json_is_false(root) )
+    MYTHROW(my::Json::Exception, ("Object must contain a boolean value."));
+  result = !json_is_true(root) ? true : false;
```
Comment (ygorshenin) [inline 3party/jansson/myjansson.cpp]:
```c++
result = json_is_true(root);
```

Comment (mgsergio) [inline 3party/jansson/myjansson.cpp]: Done.

**Comment by mgsergio** [comment]
PTAL

**Pull request merged**